config/authentication.go

```diff
@@ -0,0 +1,46 @@
+package config
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/kiali/kiali/log"
+)
+
+func AuthenticationHandler(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		statusCode := http.StatusOK
+
+		if strings.Contains(r.Header.Get("Authorization"), "Bearer") {
+			err := ValidateToken(strings.TrimPrefix(r.Header.Get("Authorization"), "Bearer "))
+			if err != nil {
+				log.Warning("Error token %+v", err)
+				statusCode = http.StatusUnauthorized
+			}
+		} else if Get().Server.Credentials.Username != "" || Get().Server.Credentials.Password != "" {
```

Comment:
minor suggestion, non blocker, but just to follow same way in other code, to not abuse of the Get() call.
```
conf := Get()
conf.Server ....
```
I know, it's just a mere decoration, but I guess people will see more clear that this comes from config instead of a Get().

But anyway, it's pure optional.

Reply:
Same in other parts of the code. Just to be consistency.

Reply:
I can make change not problem lucas, my point was that Get() just return the var so...^^